tests/scripts/perf-slices.json

```diff
@@ -0,0 +1,42 @@
+{
+    "slices" : [
+        {
+            "folders" : [
+                "Benchstones"
+            ]
```

Comment:
I can't remember if we talked about this.  When you check to see if all tests are encapsulated in a test slice, it would also be a good idea to make sure that the number of test slices in the config file matches the number of slices we create in the groovy.  You can probably do this using grep on the groovy file and looking for a constant that represents the number of slices (and use that constant to create the proper number of jobs).